### PR TITLE
Elicit fix background in dark comments

### PR DESCRIPTION
**Pull request by Discordius**
Fixes: 
image

Also fixes some random pixel-rendering artifacts you can see above (the 1px gaps between the buckets).

### CODE REVIEW

File: packages/lesswrong/components/posts/ElicitBlock.tsx [hunk 1]
```diff
@@ -187,16 +195,20 @@ const ElicitBlock = ({ classes, questionId = "IyWNjzc5P" }: {
...
   return <div className={classes.root}>
     <div className={classes.histogramRoot}>
-      {times(10, (bucket) => <div key={bucket} className={classNames(classes.histogramBucket, {
-        [classes.histogramBucketCurrentUser]: roughlyGroupedData[`${bucket*10}`]?.some(({creator}) => currentUser && creator?.displayName === currentUser.displayName)
-      })}>
+      {times(10, (bucket) => <div key={bucket} 
+        className={classNames(classes.histogramBucket, {
+          [classes.histogramBucketCurrentUser]: roughlyGroupedData[`${bucket*10}`]?.some(({creator}) => currentUser && creator?.displayName === currentUser.displayName)
+        })}
+        onMouseEnter={() => roughlyGroupedData[`${bucket*10}`]?.length && setHideTitle(true)}
```
Comment (jimrandomh) [inline packages/lesswrong/components/posts/ElicitBlock.tsx:209]: If we're making this stateful/mouse-event driven, it'd be good to take the `UsersName` components out of the tree entirely, when they're not visible.

Comment (Discordius) [reply packages/lesswrong/components/posts/ElicitBlock.tsx:209]: I don't actually think so. I am just really tired of everything event-driven that is remotely dependent on mounting and unmounting nodes breaking all the time. When I experimented with removing these, with a lot of wiggling I could get it into a state where the `onMouseLeave` trigger didn't fire, and then everything is sad. 

I guess I switched my general philosophy to "just do as much as you possibly can in CSS, unless it gets really ridiculously expensive".

Comment (Discordius) [reply packages/lesswrong/components/posts/ElicitBlock.tsx:209]: For example, see: https://github.com/facebook/react/issues/4492

File: packages/lesswrong/components/posts/ElicitBlock.tsx [hunk 2]
```diff
@@ -63,6 +65,7 @@ const styles = (theme: ThemeType): JssStyles => ({
   histogramBucket: {
     display: 'flex',
     flexGrow: 1,
+    justifyContent: 'flex-end',
```
Comment (Discordius) [inline packages/lesswrong/components/posts/ElicitBlock.tsx:68]: This fixes the issue that #3684 was supposed to solve

File: packages/lesswrong/components/posts/ElicitBlock.tsx (hunk 1 above)
Comment (jimrandomh) [reply packages/lesswrong/components/posts/ElicitBlock.tsx:223]: Oh, interesting. That React bug may explain some things about stuck-hover problems we've been getting. And I see a way to fix `useHover` to thoroughly sidestep the issue.

**Review by jimrandomh — APPROVED** [review]

**Pull request merged**